docs/core/allocation.qmd

```diff
@@ -200,6 +227,30 @@ In the future there will be 2 more optimization solves:
 
 ## Example
 
-:::{.callout-note}
-An example with figures and data will be added here after addition of [allocation output files](https://github.com/Deltares/Ribasim/issues/659).
-:::
+The following is an example of an optimization problem for the example shown [here](../python/examples.ipynb#model-with-allocation):
+
+
+```{julia}
+# | code-fold: true
+using Ribasim
+using SQLite
+
+toml_path = normpath(@__DIR__, "../../generated_testmodels/allocation_example/ribasim.toml")
+cfg = Ribasim.Config(toml_path)
+db_path = Ribasim.input_path(cfg, cfg.database)
+db = SQLite.DB(db_path)
+p = Ribasim.Parameters(db, cfg)
+close(db)
```

Comment:
We now have a Model constructor that takes a toml_path and only initializes the model. Perhaps that is easier that having to open the database yourself in this case.

```suggestion
p = Ribasim.Model(toml_path).integrator.p
```